src/init.js

```diff
@@ -138,29 +137,31 @@ export const init = async (options) => {
 			try {
 				await new ConfigApi(answers.secretKey, options.dev).validateSite(answers.name, answers.siteId);
 			} catch (err) {
-				console.log(chalk.red('Verification of siteId and secretKey failed.'));
+				console.log(chalk.red('\nSite verification failed.'));
 				console.log(chalk.red(err));
 				exit(1);
 			}
 
 			// create local directory
 			let folderName = await createDir(dir);
 
+			// determine if using org or userspace
+			let creationMethod = answers.organization == user.login ? 'createForAuthenticatedUser' : 'createInOrg';
+
 			if (options.dev) {
-				console.log(chalk.blueBright('Skipping new repo creation...'));
+				console.log(chalk.blueBright('\nSkipping new repo creation...'));
 			} else {
 				// create the remote repo
-				console.log(`Creating repository...`);
+				console.log(`\nCreating repository...`);
 				let exists = false;
 
-				await octokit.repos
-					.createInOrg({
-						org: answers.organization,
-						name: answers.name,
-						private: true,
-						auto_init: true,
-					})
-					.then(() => console.log(`${chalk.greenBright(answers.name)}\n`))
+				await octokit.repos[creationMethod]({
+					org: answers.organization,
+					name: answers.name,
+					private: true,
+					auto_init: true,
+				})
+					.then(() => console.log(chalk.cyan(`${answers.organization}/${answers.name}\n`)))
```

Comment:
`answers.organization` may show up at undefined here? Perhaps change to `${answers.organization || user.login}/${answers.name}`

Reply:
This will always have something in it since the user login is now added as an option at the end of the list:
https://github.com/searchspring/snapfu/blob/5772c9d071264ca0c55e9bf53a0a53b34a5a0688/src/init.js#L112